cpp/pipelines/collect_journal_stats.cc

```diff
@@ -202,12 +202,14 @@ void CollectMostRecentEntries(const IniFile &ini_file, MARC::Reader * const read
         const std::string ppn_type(1, ppn_and_zeder_id_and_ppn_type->second.type_);
         const std::string year_as_string(std::to_string(YearStringToShort(year)));
 
+        // data is truncated to length 8
```

Comment:
This is not a good comment.  It just states what the code below does.  What we need is a comment explaining why we need to truncate the issue string!